addons/l10n_in_withholding/views/account_move_line_views.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_move_line_tree_l10n_in" model="ir.ui.view">
+        <field name="name">account.move.line.tree.l10n.in</field>
+        <field name="model">account.move.line</field>
+        <field name="inherit_id" ref="l10n_in.view_move_line_tree_hsn_l10n_in"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='l10n_in_hsn_code']" position="after">
+                <field name="tax_ids" widget="many2many_tags" domain="[('type_tax_use', '=', 'sale')]"/>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="model_l10n_in_withholding_apply_higher_tax" model="ir.actions.server">
```

Comment:
We could remove this as we have the button on the invoice.

Reply:
It is that it will be visible anywhere.

Reply:
@shid-odoo Is not that a corner case, where he can easily change it himself on the invoice?  And with the warning, they are not going to forget to change it, so it won't be much anyways.

Reply:
> @shid-odoo Is not that a corner case, where he can easily change it himself on the invoice? And with the warning, they are not going to forget to change it, so it won't be much anyways.

@jco-odoo Yes, the user can change it on the invoice, but if there are many lines, it can be tedious. However, I’m okay with moving forward without covering it if you think it’s not necessary.

Reply:
I suggest keeping the button on the invoice and removing the server action (together with the public method on account_move_line)